microstrategy_to_s3.py

```diff
@@ -22,10 +22,17 @@
 REPORTS = {
     "2020 Bond Expenses Obligated": "85A9E0A049F06D98AF1CF3BE8CDA9394",
     "All bonds Expenses Obligated": "6B0DE57644C7C9912AAAE48392873233",
+    "2020 FDUs with Subproject and Appropriation": "97DE7BEFEE4C93C18FE27199BBBA00BF",
+    "Subprojects with total Appropriation": "01EF933E6D4FA79501CF10AF39DD8163",
+    "Open Subprojects with Budget Estimate": "E2AC0ACEF944222D0039DA9FF07BA05C",
+    "FDU Expenses by Quarter": "39EAC86D2F43664777ED6C9DBA27B43B",
+    "2020 Division Group and Unit": "221FEFD57C4B215CCB6BE2BD8DEE8CA9",
```

Comment:
New microstrategy reports that power this

Reply:
thanks for adding the link in the comments below here. I wasn't able to access the reports, but I think that I may not have access because I was prompted to enter my COA credentials. Not the first time that I've had problems accessing COA resources so all good!

Reply:
hmm yeah you're not wrong. These might have been impacted by us migrating some things. I think I'll just throw the links to these in the README.